docs/src/user-guide/quick-start/clp-text/main-page.md

```
@@ -0,0 +1,128 @@
+### Searching from the command line
+
+If you'd like to search your query from the command line, run the following command from inside the package:
+
+```bash
+sbin/search.sh '<query>'
+```
+
+To narrow your search to a specific time range:
+
+* Add `--begin-time <epoch-timestamp-millis>` to filter for log events after a certain time.
+    * `<epoch-timestamp-millis>` is the timestamp as milliseconds since the UNIX epoch.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)